src/middleware.ts

```diff
@@ -99,6 +104,15 @@ export const webhookHandler = factory.createHandlers(
         },
       });
 
+      await addBountyToDiscord({
+        username: author,
+        amount: bountyAmount,
+        pr: pr_link,
+        date: createdAt,
+        discordWh : discordWH,
+        avatar : userAvatar
+      });
+
```

Comment:
sir, if possible you can add something like this here so we can get in the logs that an error has occurred in discord part 

```javascript
      const response = await tweet({
        tweet: tweetPayload,
        accessToken: newAccessToken,
      });
      if (!response.data) {
        return c.json({ message: 'Error in tweeting but saved in notion.' });
      }
```

Reply:
hello @shrutsureja , resolved this one